src/main/java/org/folio/service/manager/export/ExportManagerImpl.java

```diff
@@ -111,22 +108,24 @@ protected void exportBlocking(ExportPayload exportPayload) {
    * @param instances list of instance objects
    * @param params
    */
-  private List<JsonObject> fetchHoldingsAndItems(List<JsonObject> instances, OkapiConnectionParams params) {
+  private List<JsonObject> fetchHoldingsAndItems(List<JsonObject> instances, List<RecordType> recordTypes, OkapiConnectionParams params) {
     List<JsonObject> instancesWithHoldingsAndItems = new ArrayList<>();
     for (JsonObject instance : instances) {
       JsonObject instanceWithHoldingsAndItems = new JsonObject();
       instanceWithHoldingsAndItems.put("instance", instance);
-      List<JsonObject> holdings = recordLoaderService.getHoldingsForInstance(instance.getString("id"), params);
-      instanceWithHoldingsAndItems.put("holdings", new JsonArray(holdings));
-      List<String> holdingIds = holdings.stream()
-        .map(jo -> jo.getString("id"))
-        .collect(Collectors.toList());
-      List<JsonObject> items = recordLoaderService.getAllItemsForHolding(holdingIds, params);
-      instanceWithHoldingsAndItems.put("items", new JsonArray(items));
-
+      if(recordTypes.contains(RecordType.HOLDINGS) || recordTypes.contains(RecordType.ITEM)) {
```

Comment:
I think it would be good to also check if the transformations are present here. So that these are no called unecessarily. (currently there is nothing preveting us from seleting record types and not adding transformations in the UI)

Reply:
Ok, good point, I will add this verification